src/prefect/environments/storage/docker.py

```diff
@@ -85,11 +87,12 @@ def __init__(
         image_tag: str = None,
         env_vars: dict = None,
         files: dict = None,
-        base_url: str = None,
         prefect_version: str = None,
         local_image: bool = False,
         ignore_healthchecks: bool = False,
         secrets: List[str] = None,
+        base_url: str = None,
+        tls_config: Union[bool, "docker.tls.TLSConfig"] = False,
```

Comment:
Do you know if objects of type `TLSConfig` are pickeable?

Reply:
Just attempted a roundtrip cloudpickle on a `TLSConfig` object and it appears to work as expected:
```
In [6]: cloudpickle.dumps(t)
Out[6]: b'\x80\x04\x95\x8b\x00\x00\x00\x00\x00\x00\x00\x8c\ndocker.tls\x94\x8c\tTLSConfig\x94\x93\x94)\x81\x94}\x94(\x8c\x0fassert_hostname\x94N\x8c\x12assert_fingerprint\x94N\x8c\x0bssl_version\x94\x8c\x03ssl\x94\x8c\x10PROTOCOL_TLSv1_2\x94\x93\x94\x8c\x06verify\x94N\x8c\x07ca_cert\x94Nub.'

In [7]: a = _

In [8]: cloudpickle.loads(a)
Out[8]: <docker.tls.TLSConfig at 0x109266588>
```